app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -37,77 +36,73 @@ class InstalledAddonDetailsFragment : Fragment() {
+    private fun bind(view: View) {
         val title = addon.translatableName.translate()
         showToolbar(title)
 
-        bindEnableSwitch(addon, view)
-        bindSettings(addon, view)
-        bindDetails(addon, view)
-        bindPermissions(addon, view)
-        bindRemoveButton(addon, view)
+        bindEnableSwitch(view)
+        bindSettings(view)
+        bindDetails(view)
+        bindPermissions(view)
+        bindRemoveButton(view)
     }
 
-    private fun bindEnableSwitch(addon: Addon, view: View) {
+    private fun bindEnableSwitch(view: View) {
         val switch = view.findViewById<Switch>(R.id.enable_switch)
         switch.setState(addon.isEnabled())
         switch.setOnCheckedChangeListener { _, isChecked ->
             if (isChecked) {
                 requireContext().components.addonManager.enableAddon(
                     addon,
                     onSuccess = {
-                        Toast.makeText(
-                            requireContext(),
-                            "Successfully enabled ${addon.translatableName.translate()}",
-                            Toast.LENGTH_SHORT
-                        ).show()
+                        showSnackBar(
+                            view,
+                            "Successfully enabled ${addon.translatableName.translate()}"
+                        )
                     },
                     onError = {
-                        Toast.makeText(
-                            requireContext(),
-                            "Failed to enabled ${addon.translatableName.translate()}",
-                            Toast.LENGTH_SHORT
-                        ).show()
+                        showSnackBar(
+                            view,
+                            "Failed to enabled ${addon.translatableName.translate()}"
+                        )
                     }
                 )
             } else {
                 requireContext().components.addonManager.disableAddon(
                     addon,
                     onSuccess = {
-                        Toast.makeText(
-                            requireContext(),
-                            "Successfully disabled ${addon.translatableName.translate()}",
-                            Toast.LENGTH_SHORT
-                        ).show()
+                        showSnackBar(
+                            view,
+                            "Successfully disabled ${addon.translatableName.translate()}"
+                        )
                     },
                     onError = {
-                        Toast.makeText(
-                            requireContext(),
-                            "Failed to disabled ${addon.translatableName.translate()}",
-                            Toast.LENGTH_SHORT
-                        ).show()
+                        showSnackBar(
+                            view,
+                            "Failed to disabled ${addon.translatableName.translate()}"
```

Comment:
nit: This should be "Failed to disable". Are these the strings you wanted to add to `feature-addons` because we can't leave them here hard-coded?

Reply:
Yes, I'm putting a PR on ac to pull this strings out as they are duplicated across apps :)

Reply:
👍

Reply:
https://github.com/mozilla-mobile/android-components/pull/5702 :)